api/envoy/admin/v3/server_info.proto

```diff
@@ -79,6 +79,14 @@ message CommandLineOptions {
     InitOnly = 2;
   }
 
+  enum DefaultsProfile {
+    // Default option: run with settings optimized for performance.
```

Comment:
Nit: we should make the "default" `DEFAULT` and leave this to Envoy to modify as needed. It's a bad idea being opinionated about the default of an enum, as this might change.

Reply:
Done